### ISSUE TITLE
27k Support

### ISSUE THREAD

**Issue by jlevy44** [perishky/meffil#19]
Hi there, does meffil offer support for the 27k array? Ideally, I'm looking to conduct a study that would combine 450k, 850k and 27k info.

Thanks!

**Comment by perishky** [comment]
No, unfortunately it doesn't.  Adding 27K support wouldn't be difficult but would require a bit of work handling slight differences between 27K and 450K control probes and adding a 27K annotation.

**Comment by jlevy44** [comment]
Yeah that does seem a bit difficult. I suppose would it be okay to preprocess the 27k data using another package and combine it with the 450?

**Comment by perishky** [comment]
Probably best to apply the same preprocessing to all samples. Seems like you could use the following function in minfi:
Ihttps://rdrr.io/bioc/minfi/man/combineArrays.html

**Comment by jlevy44** [comment]
Thanks!
